### PR TITLE
fix: rename role attribute value of fast-divider from seperator to separator

### PR DESCRIPTION
**Pull request by abhishekjakhar**
# Description

1. Changes the name of role provided to [fast-divider] from "seperator" to "separator" to prevent accessibility issues. 
2. The role "seperator" is not supported. 

**Note:** https://www.w3.org/WAI/PF/aria-1.1/roles#separator

## Motivation & context

Fixes https://github.com/microsoft/fast/issues/3462

## Issue type checklist

- [ ] **Chore**: A change that does not impact distributed packages.
- [x] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

**Is this a breaking change?**
- [ ] This change causes current functionality to break.

**Adding or modifying component(s) in `@microsoft/fast-components` checklist**

- [ ] I have added a new component
- [ ] I have modified an existing component
- [ ] I have updated the [definition file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#definition)
- [ ] I have updated the [configuration file](https://github.com/Microsoft/fast/blob/master/packages/web-components/fast-components/CONTRIBUTING.md#configuration)

## Process & policy checklist

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.
- [ ] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: sites/fast-website/src/public/index.html [hunk 1]
```diff
@@ -1,129 +1,282 @@
...
-          </site-navigation>
-          <fast-anchor class="logo" appearance="lightweight" href="/" aria-label="Home">
-            <%= require("svg/icon-brand.svg") %>
-          </fast-anchor>
-        </footer>
-      </div>
-      <background-design></background-design>
-    </fast-design-system-provider>
-  </body>
+                <fast-divider class="footer-divider" role="separator"></fast-divider>
```
Comment (abhishekjakhar) [inline sites/fast-website/src/public/index.html:241]: I did the change here, prettier is responsible for all the other changes.

Comment (chrisdholt) [reply sites/fast-website/src/public/index.html:241]: Thanks! I didn't think we had prettier running on our commit hook. Out of curiosity, did this run on the hook or did you run prettier manually?

Comment (abhishekjakhar) [reply sites/fast-website/src/public/index.html:241]: It was running when I saved the file, I can go ahead and turn off formant on save option of it is not running in pre commit hook

Comment (chrisdholt) [reply sites/fast-website/src/public/index.html:241]: I wouldn't worry about it! I think we'll set it up to run on HTML either way. :)

Only thing I noticed is that in this implementation we could probably just drop the role as `separator` is implicit on Divider.

Comment (abhishekjakhar) [reply sites/fast-website/src/public/index.html]: I removed the role and verified it, it works fine, thanks for me letting know 👍

Comment (awentzel) [reply sites/fast-website/src/public/index.html]: Thanks @abhishekjakhar for your contribution.

**Comment by chrisdholt** [comment]
Looks like there is a conflict due to the addition of prettier to support HTML. If you can resolve the conflict I'd love to get this merged!

**Comment by abhishekjakhar** [comment]
> Looks like there is a conflict due to the addition of prettier to support HTML. If you can resolve the conflict I'd love to get this merged!

I have resolved the conflicts and pushed the changes.

**Comment by abhishekjakhar** [comment]
> Looks like there is a conflict due to the addition of prettier to support HTML. If you can resolve the conflict I'd love to get this merged!

My FAST / lint_title (pull_request) check is failing, I have changed name of my PR title and my commit message also matches the PR title, I don't have any idea why it is failing, can you help me?

**Comment by chrisdholt** [comment]
Thanks @abhishekjakhar - I noticed that the check has been failing on external contributors; we'll need to look into that. Appreciate the contribution!

**Comment by abhishekjakhar** [comment]
> Thanks @abhishekjakhar - I noticed that the check has been failing on external contributors; we'll need to look into that. Appreciate the contribution!

Thank you very much! 🙏

**Pull request merged**